content/java/concepts/math-methods/terms/ceil/ceil.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.ceil()'
+Description: 'Returns the smallest (closest to negative infinity) double value that is a mathematical integer and greater than or equal to the original value.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Double'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.ceil()`** method returns the smallest (closest to negative infinity) `double` value that is a mathematical integer and greater than or equal to the original value.
+
+- The result is the same as the argument if the argument value is already equal to a mathematical integer.
+- The result is identical to the argument if the argument is `NaN`, +/- infinity, or +/- zero.
+
+## Syntax
+
+```pseudo
+Math.ceil(x)
+```
+
+- The parameter `x` is of type [`double`](https://docs.oracle.com/javase/7/docs/api/java/lang/Double.html).
+- The result is of type [`double`](https://docs.oracle.com/javase/7/docs/api/java/lang/Double.html)
+
+## Example 1
```

Comment:
@ilyasKerbal this is just a comment you can resolve.

I really like the way you separated the examples into positive vs negative numbers.